analysis_options.yaml

```diff
@@ -34,7 +34,6 @@ linter:
     - library_names
     - library_prefixes
     - non_constant_identifier_names
-    - one_member_abstracts
```

Comment:
Removed because I find it annoying and sometimes one-member abstracts are useful as interfaces.